binsync_plugin.py

```diff
@@ -0,0 +1,37 @@
+# A cross-decompiler collaboration plugin
```

Comment:
You've committed this new file in this PR, which was likely a mistake based on you copying it from the `binsync` folder, then doing `git add .`. 

Notes for the future:
- Whenever you `git add` items to the VCS, double-check what you are adding. From now on, `git add .` will get you into trouble
- Whenever you make a PR and request review, go to `Files Changed` tab and self-review that all the added/deleted code is intended 
- Consider using an IDE that manages the VCS for you. I recommend PyCharm.